environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -144,17 +155,36 @@ private ValidationResult validateEntitlement(ValidationResultBuilder validationR
                         build();
             }
         }
+        String encryptionKeyResourceGroupName = azureResourceEncryptionParametersDto.getEncryptionKeyResourceGroupName();
+        if (Objects.nonNull(encryptionKeyResourceGroupName)) {
```

Comment:
Nit: It is not a good practice to have an extra method invocation for sake of null checks. The `x == null` / `x != null`expressions are preferable in almost all cases. The major exception is when using `Objects.nonNull()` as a predicate in a stream `filter()` operation, for instance.